docker/staging/docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: '3.3'
+services:
+  coordinator:
+    image: "labgrid-coordinator"
+    volumes:
+      - "./crossbar:/opt/.crossbar"
+    tty: true
+    network_mode: "host"
+    user: "${CURRENT_UID}"
+    command: bash -c "cp /opt/.crossbar/places_example.yaml /opt/.crossbar/places.yaml &&
+      crossbar start --config /opt/labgrid/.crossbar/config.yaml"
+  client:
+    image: "labgrid-client"
+    volumes:
+      - "./client/simple-test:/simple-test"
+      - "./client/.ssh:/root/.ssh"
+    tty: true
+    stdin_open: true
+    network_mode: "host"
+    tmpfs: "/tmp"
+    # Use wait-for-it to ensure exporter service is up, as exporter is assuming exporter to
+    # Use labgrid-client r to ensure the exporter has populated the resource list in the coordinator
+    command: bash -c "set -e &&
+      cd /simple-test &&
+      /opt/wait-for-it/wait-for-it.sh 127.0.0.1:20408 &&
+      while [ -z $$(/usr/local/bin/labgrid-client r) ]; do echo 'Wait one sec on coordinator' && sleep 1; done &&
+      /usr/local/bin/labgrid-client -p example-place unlock --kick ||
+      /usr/local/bin/labgrid-client -p example-place lock &&
+      /usr/local/bin/pytest --lg-env remote.yaml -s &&
```

Comment:
Maybe add `-vv` as well.

Reply:
Good point, actually I used it while debugging so fine to leave it